src/SprykerEco/Zed/Payone/Dependency/Facade/PayoneToCalculationBridge.php

```diff
@@ -30,7 +30,7 @@ public function __construct($calculationFacade)
      *
      * @return \Generated\Shared\Transfer\QuoteTransfer
      */
-    public function recalculateQuote(QuoteTransfer $quoteTransfer)
+    public function recalculateQuote(QuoteTransfer $quoteTransfer): QuoteTransfer
```

Comment:
Please, check that methods signatures in bridges are exactly the same as in the original methods.
Check other bridges too, please.

Reply:
Fixed.